benchmarks/Smalltalk/build-image-squeak.st

```diff
@@ -0,0 +1,65 @@
+| parentDir runner loader starting |
```

Comment:
The entire what you might call scripting API and the new Filesystem classes do not exist in Squeak. Also, the default directory isn't the working dir, but the folder that contains the image. So the core of these changes just deal with finding out where the everything is and loading it.